Tests/SentryTests/Integrations/Breadcrumbs/SentryAutoBreadcrumbTrackingIntegrationTests.swift

```diff
@@ -68,7 +70,9 @@ class SentryAutoBreadcrumbTrackingIntegrationTests: XCTestCase {
         SentrySDK.setCurrentHub(hub)
         
         let crumb = TestData.crumb
+#if os(iOS)
         fixture.systemEventBreadcrumbs?.startWithdelegateInvocations.first?.add(crumb)
+#endif // os(iOS)
```

Comment:
`m`: I think this change could be responsible for `SentryAutoBreadcrumbTrackingIntegrationTests.testInstall` failing. The test succeeds for all iOS targets and fails for the other ones. The change here or in `SentryAutoBreadcrumbTrackingIntegration.m` most likely is responsible for this failing test.